README.md

```diff
@@ -738,6 +739,37 @@ Example invocation: `$ m . /lintPython/dirOfModules/makes/main`
 
 Example invocation: `$ m . /lintPython/module/cliMain`
 
+### lintWithLizard
+
+Using Lizard to check
```

Comment:
Please add a reference (link to Lizard homepage)

so people can follow the idea